src/Resources/contao/languages/en/default.xlf

```diff
@@ -1115,6 +1112,9 @@ This e-mail has been generated by Contao. You can not reply to it directly.
       <trans-unit id="MSC.logoutBT">
         <source>Logout</source>
       </trans-unit>
+      <trans-unit id="MSC.switchBT">
+        <source>Back to %s</source>
```

Comment:
Why do you not want to use "go back"?

Reply:
First of all, I figured we could show the username the link is back to. Also, I think the word "Go back" does not necessarily work for "log back in as another user". It works perfectly in german (*Zurück*) but even english (*Go back*) it's not very obvious where to "back" is. *Back to username* is a better explanation of the link.

Btw, I also considered adding both links (a real logout and a back link), not sure if we want that?

Reply:
We do not. 😄